### PR TITLE
Addition of JUnit test for Ui class

### LINKED ISSUE #24: Add JUnit test for UI class

### PR DESCRIPTION
**Pull request by Johnson-Yee**
Fix #24

### CODE REVIEW

File: src/main/java/seedu/duke/ui/Ui.java [hunk 1]
```diff
@@ -4,10 +4,9 @@
 
 public class Ui {
 
-    private static Scanner in = new Scanner(System.in);
-
     public static String getCommand() {
-        return in.nextLine();
+        Scanner userInput = new Scanner(System.in);
```
Comment (wamikamalik) [inline src/main/java/seedu/duke/ui/Ui.java]: Is there a reason why a new scanner is being declared every time a command is taken in? I believe this works the same way as our previous approach right?

File: src/test/java/seedu/duke/ui/UiTest.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class UiTest {
+
+    @Test
+    void getCommand_success() {
```
Comment (wamikamalik) [inline src/test/java/seedu/duke/ui/UiTest.java:13]: Is it necessary to have such a test? 🤔 getCommand() just calls the nextLine() function of scanner class. So by writing this test, we are essentially testing the nextLine() function which has not been written by us.

Comment (Johnson-Yee) [reply src/test/java/seedu/duke/ui/UiTest.java:13]: Initially, I added a some string functions to edit the getCommand(). However, in hindsight, the string manipulation should not be done in the ui function and I removed them. The UiTest in this case is trivial and not needed! I will close this pull request soon.

File: src/main/java/seedu/duke/ui/Ui.java (hunk 1 above)
Comment (Johnson-Yee) [reply src/main/java/seedu/duke/ui/Ui.java]: Edited it! Thank you for spotting that

**Pull request closed without merging**